Formula/docker-machine.rb

```diff
@@ -36,6 +36,43 @@ def install
     end
   end
 
+  def caveats
```

Comment:
Unless these caveats are Homebrew-specific we don't want them, really. We try to keep general usage guidelines with upstream.